style.css

```diff
@@ -1180,3 +1214,132 @@ textarea {
   color: #fff;
   background-color: #2230d2;
 }
+
+@media only screen and (min-width: 768px) {
+  .toggle-hidden1 {
+    display: none;
+  }
+
+  .toggle-hidden2 {
+    display: flex;
+  }
+
+  .headline-container1 {
+    background: url(assets/header-bg-desktop.svg);
+    position: relative;
+    width: 100vw;
+    height: 721.46px;
+    left: 34.6px;
+    top: 0;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] Although the background of the main action for the desktop version has been successfully changed :+1:, there are certain matters that require attention. The following steps can be taken to address them:

- The information within the section should be centered to align with the margins and padding of the template.
- The background image needs to be enlarged as it currently occupies only a portion of the background.
- The bottom left corner should have a rounded border in accordance with the template.

**Your design**

![Screenshot from 2023-03-29 08-00-21](https://user-images.githubusercontent.com/105117832/228544004-c674176c-60cd-411e-9def-12fded7e0e1c.png)

**Figma**

![Screenshot from 2023-03-29 08-03-49](https://user-images.githubusercontent.com/105117832/228544139-121463af-e64f-497a-afb8-4746bb2f3e99.png)

Reply:
I implemented the requested changes only the Bottom Left Corner remains in this Section.